src/diagnostics/core_diagnostics.jl

```diff
@@ -1163,3 +1163,154 @@ add_diagnostic_variable!(
     comments = "Mass of water vapor per mass of air",
     compute! = compute_husv!,
 )
+
+###
+# Implicit solver
+###
+function compute_ejac1!(out, state, cache, time)
+    FT = eltype(state)
+    device = ClimaComms.device(state)
+    first_column_jacobian = view(cache.jacobian.cache.column_matrices, 1, :, :)
+    if isnothing(out)
+        column_length = length(first(column_iterator(state)))
+        out = Array{FT}(undef, column_length, column_length)
+    end
+    ClimaComms.allowscalar(copyto!, device, out, first_column_jacobian)
+    view(out, LinearAlgebra.diagind(out)) .+= 1
+    out ./= cache.jacobian.cache.dtγ_ref[]
+end
+
+add_diagnostic_variable!(
+    short_name = "ejac1",
+    long_name = "Exact Jacobian matrix of first column",
+    standard_name = "exact_jacobian",
+    units = "",
+    comments = "Exact Jacobian matrix of tendency in first column",
+    compute! = compute_ejac1!,
+)
+
+tensor_axes_tuple(type) =
+    type <: Geometry.AxisTensor ?
+    map(axis -> typeof(axis).parameters[1], axes(type)) : ()
+
+# TODO: Move this function to ClimaCore.
+# In order for the following function to be used in Field broadcast expressions,
+# it has to be type-stable. As of Julia 1.10, this means that it needs to use
+# unrolled functions, and that its default recursion limit must be disabled.
+primitive_value_at_index(value, (row_axes, col_axes)) =
+    if isprimitivetype(typeof(value)) # same as a LinearAlgebra.UniformScaling
+        row_axes == col_axes ? value : zero(value)
+    elseif value isa Geometry.AxisVector
+        @assert isprimitivetype(eltype(value))
+        @assert length(row_axes) == 1 && length(col_axes) == 0
+        value_axes = tensor_axes_tuple(typeof(value))
+        row_axis_index = findfirst(==(row_axes[1]), value_axes[1])
+        isnothing(row_axis_index) ? zero(eltype(value)) : value[row_axis_index]
+    elseif value isa Geometry.AxisTensor
+        @assert isprimitivetype(eltype(value))
+        @assert length(row_axes) == 1 && length(col_axes) == 1
+        value_axes = tensor_axes_tuple(typeof(value))
+        row_axis_index = findfirst(==(row_axes[1]), value_axes[1])
+        col_axis_index = findfirst(==(col_axes[1]), value_axes[2])
+        isnothing(row_axis_index) || isnothing(col_axis_index) ?
+        zero(eltype(value)) : value[row_axis_index, col_axis_index]
+    elseif value isa LinearAlgebra.Adjoint
+        primitive_value_at_index(parent(value), (col_axes, row_axes))
+    else
+        sub_names = fieldnames(typeof(value))
+        sub_values =
+            MatrixFields.unrolled_map(Base.Fix1(getfield, value), sub_names)
+        nonempty_sub_values =
+            MatrixFields.unrolled_filter(x -> sizeof(x) > 0, sub_values)
+        @assert length(nonempty_sub_values) == 1
+        primitive_value_at_index(nonempty_sub_values[1], (row_axes, col_axes))
+    end
+@static if hasfield(Method, :recursion_relation)
```

Comment:
```suggestion

@static if hasfield(Method, :recursion_relation)
```
So that it's clear that this is separate from the above function.